pyproject.toml

```diff
@@ -25,7 +25,9 @@ license = "APACHE-2.0"
 python = "^3.7"
 solnlib = "^4.0.0"
 splunk-sdk = "^1.6.16"
-splunktalib = "^2.0.0"
+splunktalib = "^3.0.0b1"
```

Comment:
Can you please use splunktalib 3.0.0 release.

https://github.com/splunk/addonfactory-ta-library-python/releases/tag/v3.0.0

It's the same as 3.0.0b1, but not a beta release.

Reply:
Made the changes